java/code/src/com/redhat/rhn/manager/contentmgmt/ContentManager.java

```diff
@@ -853,6 +853,10 @@ private static void alignErrata(Channel src, Channel tgt, Collection<ErrataFilte
     /**
      * Filters given entities based on given filters.
      *
+     * Returns a Pair containing:
+     * - set of entities that are kept (not filtered out)
```

Comment:
I think this is "Left Side"

Reply:
Ok, I made it more explicit and force pushed ;)